lib/nostrum/struct/message.ex

```diff
@@ -121,19 +172,27 @@ defmodule Nostrum.Struct.Message do
   referenced_message field is not present, the backend did not attempt to fetch the message that was being replied to,
   so its state is unknown. If the field exists but is null, the referenced message was deleted.
   """
-  @type referenced_message :: t() | nil
+  @type referenced_message :: __MODULE__.t() | nil
+
+  @typedoc """
+  the thread that was started from this message, includes thread member object
```

Comment:
This docstring should match the format of the other docstrings:

```suggestion
  The thread that was started from this message, includes a thread member object
```